tests/run/cyfunction.pyx

```diff
@@ -305,6 +305,16 @@ def test_annotations(a: "test", b: "other" = 2, c: 123 = 4) -> "ret":
     def inner(x: "banana", y: b()) -> c():
         return x,y
     return inner
+    
+    
+cpdef bint test_cpdef_annotations(a: "test", b: "other" = 2, c: 123 = 4):
```

Comment:
I had to change to a less "interesting" return type here (rather than `"ret"` for the `def` version) for the C version to compile.